Runtime-Implementation-Details/Relational-Storage.md

```diff
@@ -2,33 +2,112 @@
 layout: page
 title: Relational Storage
 ---
+Relational storage backend code in Orleans is built on generic ADO.NET functionality and is consequently database vendor agnostic. The Orleans data storage layout is explained in [Runtime Tables](Runtime-Tables). Setting up the connection strings is explained in [Orleans Configuration Guide](http://dotnet.github.io/orleans/Orleans-Configuration-Guide/) and [SQL Tables](http://dotnet.github.io/orleans/Advanced-Concepts/Configuring-SQL-Tables).
```

Comment:
Hmm, these doesn't look like going to the actual pages, but to a page that links the actual pages. Moreover, the actual pages have just ellipsis instead of an actual, working connection string. I wonder that since in some examples there's a connection string that goes to `LocalDb` if the same one should be used elsewhere?

While on Gitter, it looks like many people do not look into the Nuget folder for the scripts and instead head to GitHub and take it from the latest master. Regardless even if they are running older than the current version of Nugets. This creates problems. So, it might make sense to instruct explicitly to use the script found in the Nuget package.